tests/benchdnn/dnnl_common.cpp

```diff
@@ -272,7 +272,7 @@ void init_isa_settings() {
     if (hints.get() == isa_hints_t::no_hints)
         DNN_SAFE_V(dnnl_set_cpu_isa_hints(dnnl_cpu_isa_no_hints));
     else if (hints.get() == isa_hints_t::prefer_ymm)
-        DNN_SAFE_V(dnnl_set_cpu_isa_hints(dnnl_cpu_isa_prefer_ymm));
+        dnnl_set_cpu_isa_hints(dnnl_cpu_isa_prefer_ymm);
```

Comment:
I feel that updating [this function](https://github.com/oneapi-src/oneDNN/blob/main/src/cpu/platform.cpp#L82) makes more sense. Aarch64 is fine to return `success` unconditionally.

Reply:
+1 on Dmitri's solution as it is the simplest solution.

Down the line, it might also make sense to guard the platform specific isa/isa_hint enum values in the headers. Currently, DNNL_X64 is not exposed there but we could add it to dnnl_config.h (similar to what we do for experimental features).

Reply:
Thank you for the review, I have update the code as suggested